tests/Aspire.Confluent.Kafka.Tests/OtelMetricsTests.cs

```diff
@@ -69,7 +69,7 @@ public async Task EnsureMetricsAreProducedAsync(bool useKeyed)
             ? host.Services.GetRequiredKeyedService<IProducer<string, string>>(key)
             : host.Services.GetRequiredService<IProducer<string, string>>())
         {
-            for (int i = 0; i < 100; i++)
+            for (int i = 0; i < 5; i++)
```

Comment:
The tests were taking 45 seconds each on my machine. I thought making this less would take the time away, but the time appears to be waiting for the first "Consume" to work.

```

  Standard Output: 
1/14/2025 12:24:21 AM: produced message 0
1/14/2025 12:24:21 AM: produced message 1
1/14/2025 12:24:21 AM: produced message 2
1/14/2025 12:24:21 AM: produced message 3
1/14/2025 12:24:21 AM: produced message 4
1/14/2025 12:24:21 AM: before flush
1/14/2025 12:24:22 AM: after flush
1/14/2025 12:24:22 AM: after Subscribe
1/14/2025 12:25:05 AM: consumed message 0
1/14/2025 12:25:05 AM: consumed message 1
1/14/2025 12:25:05 AM: consumed message 2
1/14/2025 12:25:05 AM: consumed message 3
1/14/2025 12:25:05 AM: consumed message 4
```